app/utils/client.js

```diff
@@ -1,7 +1,8 @@
+import Config from "react-native-config";
 import ApolloClient from "apollo-boost";
 
 const client = new ApolloClient({
-  uri: "https://apprentice-progress-api.herokuapp.com/",
+  uri: Config.PRODUCTION_API_ENDPOINT,
```

Comment:
Would it be more appropriate to have an `API_ENDPOINT` env var, which is set to the right thing depending on the environment file used? For example `.env.production` would have the production API endpoint, `.env.(local|development)` would have the local instance etc.

Reply:
Yes, so this would be better and I think I'll end up with that. BUT it comes with a lot of overhead because I'd have to have different iOS schemes, etc. which felt like a lot of overhead for one variable that's only needs change occasionally for testing.